x11-terms/xterm/files/xterm-372-musl.patch

```diff
@@ -0,0 +1,28 @@
+# Patch taken from Alpine linux [1]. Without this patch xterm won't work on
```

Comment:
This should go upstream.

Reply:
@thesamesam Made revision ebuild. Does it look alright?

Reply:
Looking good. Have you emailed DIckey with the patch and info?

Reply:
No I've not. I'll do that either today or tomorrow